OpenDreamRuntime/Objects/Types/DreamObjectException.cs

```diff
@@ -0,0 +1,44 @@
+namespace OpenDreamRuntime.Objects.Types;
+
+public sealed class DreamObjectException : DreamObject {
+    public string Name = string.Empty;
+    public string Description = string.Empty;
+    public string File = string.Empty;
+    public string Line = string.Empty;
+
+    //TODO: Match the format of BYOND exceptions since SS13 does splittext and other things to extract data from exceptions
+
+    public DreamObjectException(DreamObjectDefinition objectDefinition) : base(objectDefinition) {
+
+    }
+
+    protected override bool TryGetVar(string varName, out DreamValue value) {
+        switch (varName) {
+            case "name":
+                value = new DreamValue(Name);
+                return true;
+            case "desc":
+                value = new DreamValue(Description);
+                return true;
+            case "file":
+                value = new DreamValue(File);
+                return true;
+            case "line":
+                value = new DreamValue(Line);
+                return true;
+            default:
+                return base.TryGetVar(varName, out value);
+        }
+    }
+
+    protected override void SetVar(string varName, DreamValue value) {
```

Comment:
I haven't checked if the vars are readonly or not.

Reply:
These vars appear to be fully r/w in DM.
https://discord.com/channels/484170914754330625/487268744419344384/1191197551060336821